crates/shaders/build.rs

```diff
@@ -28,7 +28,13 @@ fn main() {
         .and_then(|p| Path::new(&p).parent().map(|p| p.to_owned()))
         .unwrap_or(PathBuf::from("../../"));
     let shader_dir = Path::new(&workspace_dir).join("shader");
-    let mut shaders = compile::ShaderInfo::from_dir(shader_dir);
+    let mut shaders = match compile::ShaderInfo::from_dir(shader_dir) {
+        Ok(s) => s,
+        Err(err) => {
```

Comment:
In theory, we should coalesce issues from multiple shaders.

In practise, you'll only be working on one shader at a time, so it should be fine. I guess if you modify an include in an incompatible way, this could be an issue